### ISSUE TITLE
Usando VSCode like a pro

### ISSUE THREAD

**Issue by barbunzel** [horchatajs/charlas#41]
**Título:** Usando VSCode _like a pro_

**Descripción:** Una charla para aprender un poco de cómo trabajan en VSCode los power users.

**Ponente:** ¿Alguien?

**Email (si no está en el perfil de github):**

**Tipo:** lightning talk (15 minutos) | corta duración (25 minutos) | larga duración (40 minutos)

Quisiera que alguien compartiera su experiencia. Yo no soy tan power user en VSCode. Me sé algunas cosas, pero hay otras que no las ocupo al día a día, así que no soy el mejor para compartirlo. Pero sería súper interesante ver cómo lo ocupa alguien que se sepa muchos shortcuts y cosas así. Por ejemplo, hay plugins como [WakaTime](https://wakatime.com/) para monitorear tu productividad.

**Comment by ricardoerl** [comment]
Yo también estaría interesado en una charla de productividad con VSCode. 🙌

**Comment by barbunzel** [comment]
Tres Doritos después... Tal vez puedo hablar un poco de esto en alguno de los eventos en los próximos meses. No soy tan pro, pero nadie ha propuesto hablar de esto. :( Si necesitan alguna charla por ahí, podría ser otra vez a lo Zordon. :D

**Comment by ricardoerl** [comment]
@dieguito151 hagamosla otra vez a lo Zordon 🙌 ¿Por donde podriamos enforcar la charla?  ¿Plugins que no deben faltar? ¿Debugging con VSC?

**Comment by barbunzel** [comment]
Mmm, yo pensaba más así como shortcuts que tiene VSCode y más cosas que podés hacer. Pero tal vez se puede hacer algo más específico.

**Comment by luisquezada-net** [comment]
Hola @ricardoerl y @luisdiegohernandez - les presento a Mauricio Reyes @reyesmarq quien esta interesado en presentar este tema. Avisennos como podemos hacer o cuales son los next steps. Gracias!

**Comment by barbunzel** [comment]
@luisquezada-net @reyesmarq hey, ¡qué cool! Yo pensaba darla el 24 de octubre porque nadie más estaba disponible, pero con gusto podés agarrar mi espacio. Estoy seguro que vos vas a poder presentar cosas más interesantes de las que yo podría presentar, y así también me da oportunidad de pensar en otro tema. :)

**Comment by reyesmarq** [comment]
Hola un gusto. Gracias por la oportunidad, espero poder compartir cosas que beneficien a todos.

**Comment by ricardoerl** [comment]
Un gusto @reyesmarq, muchas gracias por el interes en apoyar. Entiendo @luisdiegohernandez que le cederias el espacio del 24 de octubre a Mauricio? En ese caso solo necesitaria la confirmacion de disponibilidad de Mauricio y su correo para poder enviarle mas detalles.

Muchas gracias @luisquezada-net por la presentación.

**Comment by barbunzel** [comment]
Sí, claro, si @reyesmarq tiene tiempo ese día, por mí está bien. Podemos encontrar otro espacio para otro tema. O si no puede ese día, siempre puedo ocupara ese espacio. Me avisás cualquier cosa @ricardoerl. :)

**Comment by reyesmarq** [comment]
@luisdiegohernandez Ese dia me parece bien 24 Octubre, no hay problema. Solo confirmame la hora y con gusto.

**Comment by reyesmarq** [comment]
@luisdiegohernandez mi correo es reyes.marq@gmail.com para detalles o algun otro recurso que te gustaria nos coordinaramos.
